src/Raven.Server/Documents/Handlers/Debugging/MemoryDebugHandler.cs

```diff
@@ -475,6 +477,7 @@ internal class MemoryInfo
             public string LowMemLastFiveMinute { get; set; }
             public string HighMemSinceStartup { get; set; }
             public string LowMemSinceStartup { get; set; }
+            public string LockedMemory { get; set; }
```

Comment:
I wonder if adding "encryption" to the name wouldn't make it more clear e.g. `EncryptonLockedMemory`

Reply:
done